app/javascript/lib/table-item-anchors.js

```diff
@@ -8,6 +8,10 @@ document.addEventListener("DOMContentLoaded", function () {
   });
   [...document.querySelectorAll("table[data-anchor-rows] tr[id] a")].forEach(
     function (link) {
+      if (link.getAttribute('href')[0] === '#') {
```

Comment:
Should probably adjust this check to be part of the `querySelector`, eg `a:not([href='#'])`

Reply:
Good shout, I forget how powerful CSS queries are, they can even handle "href _starts with_ a hash"